components/molecule/inputField/src/index.js

```diff
@@ -2,35 +2,50 @@ import React from 'react'
 import PropTypes from 'prop-types'
 
 import MoleculeField from '@s-ui/react-molecule-field'
-import AtomInput from '@s-ui/react-atom-input'
+import AtomInput, {inputStates} from '@s-ui/react-atom-input'
 
 const getErrorState = (success, error) => {
   if (success) return false
   if (error) return true
 }
 
+const getState = (success, error, alert) => {
+  if (success) return inputStates.SUCCESS
+  if (error) return inputStates.ERROR
+  if (alert) return inputStates.ALERT
+}
+
 const MoleculeInputField = ({
   id,
   label,
   successText,
   errorText,
+  alertText,
   helpText,
   inline,
   onChange,
   ...props
 }) => {
   const errorState = getErrorState(successText, errorText)
+  const inputState = getState(successText, errorState, alertText)
```

Comment:
check out https://github.com/ryanmcdermott/clean-code-javascript#functions, maybe you agree

Reply:
Totally! Thanks! 🚀

Reply:
Fixed, thanks a lot!